asdf/_tests/_issues/test_1553.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+import asdf
+
+
+def test_1553(tmp_path):
```

Comment:
Use a descriptive name for this test.

Reply:
This follows the convention set up in: https://github.com/asdf-format/asdf/pull/1537

Reply:
I completely disagree with this approach. Test names should be descriptive of what is going on in a test. That way you have an idea of what is broken when a test fails without having to go and read a docstring and then some issue on github.